sky/backends/backend_utils.py

```diff
@@ -3342,6 +3355,8 @@ def _update_records_with_resources(
     def _refresh_cluster_record(cluster_name):
         record = _refresh_cluster(cluster_name,
                                   force_refresh_statuses=force_refresh_statuses,
+                                  retry_if_missing=(force_refresh_statuses
+                                                    is not None),
```

Comment:
Can we just check `refresh` directly since that's how `force_refresh_statuses` is derived?

Reply:
Maybe we should just set to True here